examples/flutter_search/lib/models/entities.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:equatable/equatable.dart';
+
+class SearchResult extends Equatable {
+  final List<SearchResultItem> items;
+  final bool isPopulated;
+  final bool isEmpty;
+
+  SearchResult({this.items, this.isPopulated, this.isEmpty})
+      : super([items, isPopulated, isEmpty]);
+
+  static SearchResult fromJson(Map<String, dynamic> json) {
+    final items = (json['items'] as List<dynamic>)
+        .map((dynamic item) =>
+            SearchResultItem.fromJson(item as Map<String, dynamic>))
+        .toList();
+    return SearchResult(
+      items: items,
+      isPopulated: items.isNotEmpty,
+      isEmpty: items.isEmpty,
+    );
+  }
+
+  @override
+  String toString() =>
+      'SearchResult { items: ${items.toString()}, isPopulated: $isPopulated, isEmpty: $isEmpty }';
+}
+
+class GithubUser extends Equatable {
+  final String login;
+  final String avatar_url;
```

Comment:
the dart coding guidelines say to use camelCase for member variables so this should be `avatarUrl`